### ISSUE TITLE
Self recorded demos doesn't work

### ISSUE THREAD

**Issue by toorisrael** [LaihoE/demoparser#212]
Demos recorded by client using command "record demoname" don't work in current version:
```
thread '<unnamed>' panicked at D:\a\demoparser\demoparser\src\parser\src\first_pass\frameparser.rs:223:26:
index out of bounds: the len is 0 but the index is 0
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread '<unnamed>' panicked at D:\a\demoparser\demoparser\src\parser\src\parse_demo.rs:69:20:
a scoped thread panicked
Traceback (most recent call last):
  File "myscriptname.py", line 31, in <module>
    mydisconnectevents = parser.parse_event("player_disconnect")
pyo3_runtime.PanicException: a scoped thread panicked
```
For example this demo works in 0.17.0

**Comment by LaihoE** [comment]
Thanks for bringing this up. Should be fixed in 0.32.2 (building atm)

**Comment by toorisrael** [comment]
Thanks it works now
